bindings/python/mata.pyx

```diff
@@ -149,16 +149,19 @@ cdef class Nfa:
     #  potentially create some kind of Factory/Allocator/Pool class, that would take care of management of the pointers
     #  to optimize the shared pointers away if we find that the overhead is becoming too significant to ignore.
     cdef shared_ptr[mata.CNfa] thisptr
-    cdef alphabet
 
-    def __cinit__(self, state_number = 0, alphabet = None):
-        """Constructor of the NFA
+    def __cinit__(self, state_number = 0, Alphabet alphabet = None):
+        """
+        Constructor of the NFA.
 
         :param int state_number: number of states in automaton
         :param Alphabet alphabet: alphabet corresponding to the automaton
         """
-        self.thisptr = make_shared[CNfa](mata.CNfa(state_number))
-        self.alphabet = alphabet
+        cdef CAlphabet* c_alphabet = NULL
+        cdef COrdVector[State] state_set
```

Comment:
What is the state set here? Missing the comment/better naming.

Reply:
OK, maybe call it `empty_default_sets` instead then?

Reply:
Sounds better. Will do.